CONTRIBUTING.md

```diff
@@ -1,109 +1,143 @@
+
+```bash
+TENDERLY_KEY=<your access key>
+TENDERLY_ACCOUNT=<your account/organization name>
+TENDERLY_PROJECT=<your project name>
 ```
 
-For running the integration test suite you need to run the app. You can either run again `yarn dev` which will be a bit more resource intensive as the app will be build on the fly or run against a static build via `yarn build:static` & `yarn serve:static`.
+For running the integration test suite, you’ll need to have the application running locally in a separate terminal. You may choose to either run it in development mode via `yarn dev` or against a static production build via `yarn build:static` & `yarn serve:static`. The caveat to running in development mode is that it will be more resource-intensive, since the application will be built on the fly.
 
-```sh
+```bash
 # open interactive cypress test suite
 yarn test:open
+
 # run all tests in headless mode
 yarn test:headless
 ```
 
-## Run on a fork
+## Environment Variables
 
-You can run the ui against a forked network similar to what the tests do which allows you to play around on the ui without spending actual funds.
-To enable forks in the ui, you have to run the following commands in console.
+Some environment variables can be adjusted to suite your needs during development, depending on the scenarios you want to test or build in.
 
-```js
-localStorage.setItem('forkEnabled', 'true');
-localStorage.setItem('forkBaseChainId', 1); // the networkId you are forking
-localStorage.setItem('forkNetworkId', '3030'); // the networkId on the fork
-localStorage.setItem('forkRPCUrl', <rpcurl>);
+```bash
+# setting the environment to 'staging' will enable testnet markets, disabling governance, staking, and production markets
+NEXT_PUBLIC_ENV=prod
+
+# you can also disable staking & governance by altering
+NEXT_PUBLIC_ENABLE_GOVERNANCE=true
+NEXT_PUBLIC_ENABLE_STAKING=true
 ```
 
-As localeStorage is not observed you need to **reload** after setting the parameters.
-Now the market selection should show forked markets for all the markets that run on `forkBaseChainId`.
-To do actual transactions on the fork you need to setup your wallet to use the same `rpcurl` you provided as `forkRPCUrl`.
+## Running Against a Chain Fork
 
-## Token addition
+You can run the UI locally against a forked chain network, similar to what the tests do with Tenderly. This will allow you to build and interact with the UI without spending actual funds. This is very useful for testing many protocol scenarios.
 
-To add a new token to the app, all you have to do is adding a svg token icon inside `public/icons/tokens` please make sure the svg name equals the `lowercase` onchain `symbol`.
-If you want a custom name to appear alongside the icon you can alter `src/ui-config/reservePatches.ts` accordingly.
+First, you’ll need to create a fork. This can be done with a few different tools. We have set up a repo to help out with this [here](https://github.com/aave/tenderly-fork).
```

Comment:
Should we keep this statement in or remove it?

Reply:
New statement:

`We suggest following the one created by Bored Ghost Labs [here](https://github.com/bgd-labs/aave-tenderly-cli).`